test/TextToBinary.cpp

```diff
@@ -538,4 +540,21 @@ TEST(AssemblyContextParseMessages, Errors) {
   spvDiagnosticDestroy(diag);
 }
 
+TEST(CreateContext, ValidEnvironment) {
+  EXPECT_THAT(spvContextCreate(static_cast<spv_target_env>(99999999)),
```

Comment:
GCC fails to compile this since the 99999999 value is out of range of the underlying type.  GCC might be using an 8 bit or 16 bit int for the enum.
Suggest using -1 as the invalid value.